UKHomeOffice/sgmr-service
# SPCR-186 Add cookie component and logic to not track GA

### AC / Description of changes
- Add logic for cookie banner to show/hide
- Add function to set GA cookies to false if user selects 'decline' or true if user selects 'accept'
- Set default GA cookie state to false

### To Test

- Open GA
- View real time
- Load spcr locally
- click around
- > you should NOT see traffic recorded
- select accept to cookies
- click around
- > you SHOULD see traffic recorded
- delete the localstorage cookie state
- refresh the page
- click a new page
- > you should see the cookie banner again
- select decline
- > you should NOT see traffic recorded


### Notes

---
* remember to merge to feature branches not master *
